cross-modal-search/Dockerfile

```diff
@@ -2,12 +2,16 @@ FROM jinaai/jina:2.0.8-perf
 
 RUN apt-get -y update && apt-get install -y git wget gcc unzip libmagic1
 
-COPY . /workspace
 WORKDIR /workspace
+COPY ./requirements.txt requirements.txt
+
+RUN pip3 install torch==1.9.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
 
 # install the third-party requirements
 RUN pip install git+https://github.com/openai/CLIP.git click && pip uninstall -y jina && pip install -r requirements.txt && pip uninstall -y dataclasses && python -c "import clip; clip.load('ViT-B/32')"
```

Comment:
remove `pip uninstall -y jina`

Reply:
Addressed, please see [here](https://github.com/jina-ai/examples/pull/684/commits/b941d959c228d88758aff43dfcf2a7004965f2db) for the change, thank you.